### PR TITLE
Small update of the docs

### PR DESCRIPTION
**Pull request by grittygrease**
- add the remote and auth_keys fields to the config
- fix whitespace

### CODE REVIEW

**Comment by kisom** [comment]
LGTM.

File: doc/api.txt [hunk 1]
```diff
@@ -355,10 +355,14 @@ and currently have only one field:
           * default: the default profile to use for requests that do
             not explicitly set a profile.
           * profiles: a dictionary of signing profiles.
+        * auth_keys: a dictionary containing authentication keys
```
Comment (lziest) [inline doc/api.txt]: slightly off identation.

Comment (lziest) [inline doc/api.txt]: nvm. context reveals I was wrong.

**Comment by lziest** [comment]
LGTM

**Pull request merged**